templates/cf-jobs.yml

```diff
@@ -694,6 +694,7 @@ properties:
 
   router:
     enable_ssl: (( merge || nil ))
+    enable_routing_api: (( merge || true ))
```

Comment:
@Amit-PivotalLabs we added this in manifest template to control `enable_routing_api` property on both gorouter and cloud_controller_ng jobs. Otherwise we end up changing the property defaults in two spec files for each jobs, which can get out of sync.